views/season.tmpl

```diff
@@ -1,3 +1,15 @@
+{{define "title"}}{{.PageContext.ShortName}} | {{with .PageData}}{{.Season.Title}}{{end}}{{end}}
+
+{{define "open-graph"}}
+	<meta name="twitter:card" content="summary"/>
+	<meta name="twitter:site" content="@{{.PageContext.MainTwitter}}"/>
```

Comment:
The indentation here is too much not sure any indentation is needed on any of the open graph blocks